.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "standard.enable": false
+}
```

Comment:
Is this a setting that can be set outside of the repo? I'd rather add .vscode to a .gitignore than add additional settings / configuration files to the project for a specific editor